build.sh

```diff
@@ -9,215 +9,202 @@ usage() {
   echo -e "\tz690a_ddr5     - build Dasharo image compatible with MSI PRO Z690-A (WIFI)"
   echo -e "\tz790p_ddr4     - build Dasharo image compatible with MSI PRO Z790-P (WIFI) DDR4"
   echo -e "\tz790p_ddr5     - build Dasharo image compatible with MSI PRO Z790-P (WIFI)"
+
   echo -e "\tvp66xx         - build Dasharo for Protectli VP66xx"
   echo -e "\tvp46xx         - build Dasharo for Protectli VP46xx"
   echo -e "\tvp2420         - build Dasharo for Protectli VP2420"
   echo -e "\tvp2410         - build Dasharo for Protectli VP2410"
   echo -e "\tV1210          - build Dasharo for Protectli V1210"
   echo -e "\tV1410          - build Dasharo for Protectli V1410"
   echo -e "\tV1610          - build Dasharo for Protectli V1610"
+
   echo -e "\tapu2           - build Dasharo for PC Engines APU2"
   echo -e "\tapu3           - build Dasharo for PC Engines APU3"
   echo -e "\tapu4           - build Dasharo for PC Engines APU4"
   echo -e "\tapu6           - build Dasharo for PC Engines APU6"
+
   echo -e "\toptiplex_9010  - build Dasharo compatible with Dell OptiPlex 7010/9010"
+
   echo -e "\tqemu           - build Dasharo for QEMU Q35"
   echo -e "\tqemu_full      - build Dasharo for QEMU Q35 with all menus available"
+
+  echo -e "\tv540tu         - build Dasharo for Novacustom V540TU"
+  echo -e "\tv560tu         - build Dasharo for Novacustom V560TU"
+  echo -e "\tns5x7x         - build Dasharo for Novacustom NS5x/NS7x"
+  echo -e "\tnv4x_adl       - build Dasharo for Novacustom NV4x 12 gen"
+  echo -e "\tnx5x7x         - build Dasharo for Novacustom Nx5x/Nx7x"
+  echo -e "\tnv4x_tgl       - build Dasharo for Novacustom NV4x 11 gen"
+
+  echo "Used environmental variables:"
+  echo -e "\tCB_REVISION          - build Dasharo coreboot from a given revision name"
```

Comment:
I'm not sure if this should be in this script, if it only checks out a specific rev. For example what happens if it checks out to a revision that doesn't have this script? It may stay in memory and finish the run, but after it finishes the build.sh script may be different and not work the same way if someone calls it again

Reply:
That is true. The script is a part of the repository which it would checkout to a different version itself. I wonder if using the script as a universal way of building any version of Dasharo would be possible without making changes in historic revisions...

Reply:
Maybe it would be possible to get the ID of the current commit `git rev-parse HEAD` and restore it after building? This sounds like a cheap solution though...